cmd/crdvalidator/main.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+
+	"github.com/operator-framework/rukpak/cmd/crdvalidator/internal/handlers"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+const defaultCertDir = "/etc/admission-webhook/tls"
+
+func init() {
+	log.SetLogger(zap.New())
+}
+
+func main() {
+	entryLog := log.Log.WithName("crdvalidator")
+
+	// Setup a Manager
+	entryLog.Info("setting up manager")
+	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
+	if err != nil {
+		entryLog.Error(err, "unable to set up overall controller manager")
+		os.Exit(1)
+	}
+
+	entryLog.Info("setting up webhook server")
+	hookServer := mgr.GetWebhookServer()
+
+	// Point to where cert-mgr is placing the cert
+	hookServer.CertDir = defaultCertDir
+
+	// Need manager to have scheme of CRDs
+	utilruntime.Must(apiextensionsv1.AddToScheme(mgr.GetScheme()))
```

Comment:
nit: Can we setup the scheme before instantiating the manager instance, and handle the error returned vs. relying on the utilruntime.Must (which I believe panics under-the-hood). You would then be able to pass the built up scheme variable to manager.New in the manager.Options (e.g. manager.Options{ Scheme: scheme })

Reply:
So the newest commit will be adding this fix. One thing to ask is if we want to stop using `utilruntime.Must` inside of the `plain` provisioner as well for a follow-up:

https://github.com/operator-framework/rukpak/blob/455aead0f9a3687bdee334d27483ac23d1219a4e/internal/provisioner/plain/main.go#L50-L52

Reply:
Yeah that seems like a reasonable cleanup PR in my eyes.